web/api/web_api_v1.c

```diff
@@ -1190,6 +1190,27 @@ int web_client_api_request_v1_anomaly_event_info(RRDHOST *host, struct web_clien
     freez(s);
     return HTTP_RESP_OK;
 }
+
+int web_client_api_request_v1_ml_info(RRDHOST *host, struct web_client *w, char *url) {
+    (void) url;
+
+    if (!netdata_ready)
+        return HTTP_RESP_BACKEND_FETCH_FAILED;
+
+    char *s = ml_get_host_runtime_info(host);
+    if (!s)
+        s = strdupz("{\"error\": \"json string is empty\" }\n");
```

Comment:
Hi @vkalintiris ! I assume this is expected by and handled by the consumer of the endpoint? I can't remember if in other endpoints we state that the json is empty with an error message, instead of just empty response ?

Reply:
I'll update this to send an empty response.